events/messageCreate.js

```diff
@@ -1,28 +1,25 @@
-const config = require(`../config.json`)
+const config = require(`../config.json`);
 module.exports = (client, message) => {
-
     //Ignore all bots
     if (message.author.bot) return;
 
     //Prefix
-    let prefix = config.prefix
+    let prefix = config.prefix;
 
     //Ignore messages not starting with the prefix (in config.json)
-    if (message.content.indexOf(prefix) !== 0) return;
-  
+
+    // if (message.content.indexOf(prefix) !== 0) return;
+    if (message.content.startsWith(prefix)) return;
```

Comment:
This is supposed to return when the content doesn't have the prefix

Reply:
Fixed it.